controlplane/kubeadm/internal/workload_cluster_conditions.go

```diff
@@ -326,27 +344,59 @@ func compareMachinesAndMembers(controlPlane *ControlPlane, members []*etcd.Membe
 				Reason:  controlplanev1.KubeadmControlPlaneMachineEtcdMemberNotHealthyV1Beta2Reason,
 				Message: fmt.Sprintf("Etcd doesn't have an etcd member for Node %s", machine.Status.NodeRef.Name),
 			})
+
+			// Instead, surface there is a machine without etcd member on kcp for the Available condition
+			// only if the machine is not deleting and the node exists by more than two minutes
+			// (this prevents the condition flick during scale up operations).
+			// Note: Two minutes is an arbitrary interval where we expect the system to detect the new etcd member on the machine.
+			if machine.DeletionTimestamp.IsZero() {
+				newNode := false
```

Comment:
Should we also consider this a newNode if it's not in the list?

I would also rename this to oldNode. Just to get rid of one negation